tests/wpt/mozilla/meta/MANIFEST.json

```diff
@@ -9717,6 +9729,11 @@
      {}
     ]
    ],
+   "mozilla/referrer-policy/generic/subresource/mozresource.pyc": [
```

Comment:
Ugh. I should probably fix this script to ignore pyc files. Should be fixed now!